android/src/main/java/com/tailscale/ipn/util/TSLog.kt

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+package com.tailscale.ipn.util
+
+import android.util.Log
+import libtailscale.Libtailscale
+
+object TSLog {
+  var libtailscaleWrapper = LibtailscaleWrapper()
```

Comment:
private?

Reply:
this can't be private because we use it in TimeUtilTest